README.md

```diff
@@ -35,7 +35,7 @@ while let Some(record) = iter.next() {
   println!("{}", record);
 }
 ```
-See crate documentation on available API.
+See [crate documentation](https://sunchao.github.io/parquet-rs/master/parquet/) on available API.
```

Comment:
Should this link point to gh-pages branch?

Reply:
Hmm... you mean `https://github.com/sunchao/parquet-rs/tree/gh-pages`?

Reply:
All good. I confused it with actual code in gh-pages. Link works, no need to change.

Reply:
I think you can actually use sunchao.github.io/parquet-rs/master. There is an index.html page with redirect.

Reply:
Ah OK. I'll change to `https://sunchao.github.io/parquet-rs/master` then.